### PR TITLE
Documentation updates to match recent code updates

### PR DESCRIPTION
**Pull request by sumul**
This updates the Typography and Colors guides and graphics, which was an outstanding to-do after we updated the core Phenotypes colors for accessibility improvements. Also a tiny change to our Fractal theme to remove the custom highlight color, which was not great. 

This does not introduce any library code changes and should not affect any apps using components or styles.

### CODE REVIEW

**Review by orjames — APPROVED** [review]
Looks good! Just one comment

File: fractal_assets/css/theme.css [hunk 1]
```diff
@@ -218,8 +218,8 @@ a:active {
     width: 12px;
 }
 
-::selection {background-color: #005eb0; }
-::moz-selection {background-color: #005eb0; }
+/* ::selection {background-color: #005eb0; }
+::moz-selection {background-color: #005eb0; } */
```
[the remark this answers is not in the document]
Comment (NickDunkman) [reply fractal_assets/css/theme.css:222]: @orjames FYI this is just styles for the fractal site -- doesn't actually affect the phenotypes stylesheet.

[the remark this answers is not in the document]
Comment (NickDunkman) [reply fractal_assets/css/theme.css:222]: related commit message: "Removed custom highlight color (looked terrible)."

[the remark this answers is not in the document]
Comment (sumul) [reply fractal_assets/css/theme.css:222]: Yep!

**Review by NickDunkman — APPROVED** [review]
🚀

**Pull request merged**